### ISSUE TITLE
[Screen reader - ADF - Add connection] Screen reader announces 'empty document' on Azure data factory extension dialog window.

### ISSUE THREAD

**Issue by Shalini3457** [microsoft/AzureStorageExplorer#3845]
**User Experience:**
Screen reader user will be affected if screen reader doesn't provide name of dialog window as user won't get to know about such dialog screen.

**Test Environment:**
Application Version: 1.16.0-dev
Build Number: 20200921.1
Screen reader: Narrator, NVDA

**Pre-Requisite:**
Navigate to the storage expoler.exe and select it  

**Repro Steps:​**
- Launch Azure Storage Explorer app.
- Navigate and select Add data factory control at left panel.
- Now, navigate to 'Connection' control and select '+New' option on Azure data factory window.
- Verify dialog window after selecting +New option.

**Actual Result:**
Screen reader announces 'empty document' instead of 'Azure Data Factory Extension' dialog window after selecting '+New' option under Connection control of Azure data factory extension dialog window and due to this, screen reader will be confused as they will consider there must be empty document for no use.

**Expected Result:**
Screen reader should announce 'Azure Data Factory Extension' dialog window after selecting '+New' option under Connection control of Azure data factory extension dialog window so that screen reader user could get to know about such dialog window.

**Note:**
- Screen reader doesn't announce Azure data factory window after selecting visual control azure data factory at left panel.
- NVDA also announces 'empty document' instead of 'Azure Data Factory Extension' dialog window.

[MAS1.3.1_Add connection.zip](https://github.com/microsoft/AzureStorageExplorer/files/5641392/MAS1.3.1_Add.connection.zip)

**Comment by MRayermannMSFT** [comment]
This should be fixed by ADF team.
